.travis.yml

```diff
@@ -0,0 +1,14 @@
+services:
+  - docker
+env:
+  global:
+  - BOARD_ID=5ef102923967ff06c7504e0a
+  - FLASK_APP=app
+  - FLASK_ENV=development
+  - secure: "azmepD9k5MyQeNX8fhFGiZBGMcMBS394BsgTQtPkziWm6ABXD6QuK5vYmSHyLx4HxnPM6qS/ySxPxply2Lbf/bNj/uY3NafuIj8q2b8ZC9DtGkemE8QuROmoTIL57TayFAjLfbuPhDA/cSidMx/alup56C6SoPP+O5RWrX/XzaHggx8sRGExnfk1rSnmYFbuprESy7L3TJAyrM+7/Ww2Q0iuphTtPUp4/JZyecZL6cyYSI6pS2NRuUSHqbozG6SWZn0/PtnMMhhDc0tjZW6w5FB3Jwwnfrz9Zhis4MFzaQWeCdhhsFw6jL2U59a9eJZRVPoEnBPBiBkVLDZCjH926Emun/9cdA+nA6vj2YBgwT2jAoLUPkwok+OJQVPp34sOxrrLDG6mXXzvVyxSZ5ZLLUpEbZrS6V4KOXS4leShWiFL4oYF99bK5tVIj/81G8u1PYkVGVf1Hb6DMEXHTHHtr+a9nCgJ2WNbR8efyfd6WDBzoTxsglfJZvTCej0FOcyEHSJx9heA3Nc4OAdOWmWH6noYIn6hYSAbWsFC6Z3x7NwrvANBOse1bgWlOa2J6HJCKeI9J07OEH73ovgEVtUT0sCctGall5w5RONg4HFrmiyMjteV2kGbSBsYG1mi6cDDGBU325YUIMY1iF35FfteOc2EyJKqiqmuxEbxk4auGrw="
+  - secure: "ftig8aPk8avtgzOJUHkt5aQm6iwjC1gYtxPBIVO2fnjCXqfes1HzYEKJeygO7v9BTJ1fa5k4BX0hH09waKD23eLq3+U83DlfO8pbM0jVFoO3qc8YwUYF2MDaUIDvjWv05gI5mpHCeorYnp9FjBOKeaSbEbZuxO9p/54xyrFe3tXrBzv3OFYLzbjw+6NH8sXWsLIiF8oBZolLrPp14fpzIC4hd5ogpP3OqHtnKGkm/gDhQct+pl72laBT/GuWpcf0ul/te5WLfo1SJXuGZpi5xCvWL25u/HCGVx1HGwJWtRLK77dsVpPYOnzmz6HnJ990CYu+og4s/5LQzWDvx1TdOMlYy1k5BWRBJyF2cvc91SBl0uK7grkWS4pKSn0QM0zhEc7JtrdwuebflxjinLC1T3o2MWg8bwbtCKyvKTG9ZdVtKKdgY6FdEt91AUlE7DtglwtstAMUnkfeHeQRIHIlLvptsAwoFZxHOh0uiVM3h7hpMl41eElKuFivwi0P2TDcKuNCg4ZCgL1Styryust98e8qBXAttVDgvJf76Hbn7vPWMP59gYjrBvrvng1DYgwXVgS7Pf/JkO1DvDozSk9Ll1Q/yoNXj4WRimXuCzuRb4yb/u4QL3Az7iDdStC/JMyI6as3ZrwsIDMf7JmW7f7XTEpb0YCOdBIEE08mfpP/ByE="
+script:
+- docker build --target test --tag todo-app:test .
+- docker run -e API_KEY=$API_KEY -e TOKEN=$TOKEN -e BOARD_ID=$BOARD_ID -e FLASK_APP=$FLASK_APP -e FLASK_ENV=$FLASK_ENV todo-app:test test_e2e.py
+- docker run todo-app:test test_app.py
+- docker run todo-app:test test_indexViewModel.py

```

Comment:
This obviously doesn't scale well if you keep adding test files - you would want to put them in a folder.